src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -1,6 +1,7 @@
 package frc.team2412.robot;
 
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.revrobotics.ColorSensorV3;
```

Comment:
I don't think this import is needed anymore.

Reply:
forgot to remove